src/assets/CollectiblesController.test.ts

```diff
@@ -48,9 +48,13 @@ const DEPRESSIONIST_CLOUDFLARE_IPFS_SUBDOMAIN_PATH = getFormattedIpfsUrl(
 /**
  * Setup a test controller instance.
  *
- * @returns A collection of test controllers and stubs
+ * @param options - Controller options.
+ * @param options.includeOnCollectibleAdded - Whether to include the "onCollectibleAdded" parameter.
+ * @returns A collection of test controllers and stubs.
  */
-function setupController() {
+function setupController({
+  includeOnCollectibleAdded = true,
```

Comment:
total nit, but we could default to false and then `should add collectible and collectible contract even when "onCollectibleAdded" is unset` is covered by `should add collectible and collectible contract`. Then setup controller in `should call onCollectibleAdded callback...` tests could take `includeOnCollectibleAdded` set to true and be renamed `should call onCollectibleAdded callback correctly when ___ and onCollectibleAdded  parameter was passed`. Not really a big deal, but maybe a bit more intuitive

Reply:
Hmm. Good point, that does sound better